vtex-apps/store-components
# Feature/gallery zoom

#### What is the purpose of this pull request?

Add the option to zoom products in a gallery from de product-details page.

#### What problem is this solving?

The need of a visualization of a product without any distractions.

#### How should this be manually tested?
[workspace](http://fixzoom--storecomponents.myvtex.com)
[storefront](http://fixzoom--storecomponents.myvtex.com/admin/cms/storefront)
(select a product, click in product-details inner-blocks and select product images. An option with the zoom configurations must appear.)

Workspaces with zoom configured:

[inpage wks](https://inpage--storecomponents.myvtex.com)
[gallery wks](https://galleryzoom--storecomponents.myvtex.com/)
[no zoom wks](https://nozoom--storecomponents.myvtex.com/)

#### Screenshots or example usage

#### Types of changes
